saltstack/hortonworks/salt/ccmv2-inverting-proxy-agent/init.sls

```diff
@@ -0,0 +1,6 @@
+install_ccmv2-inverting-proxy-agent:
+  cmd.script:
+    - name: salt://ccmv2-inverting-proxy-agent/install_ccmv2.sh
+    - output_loglevel: DEBUG
+    - timeout: 9000
+    - failhard: True
```

Comment:
file.managed source accepts urls. I think that would be a more elegant solution
https://docs.saltstack.com/en/latest/ref/states/all/salt.states.file.html#salt.states.file.managed
```
source

    The source file to download to the minion, this source file can be hosted on either the salt master server (salt://), the salt minion local file system (/), or on an HTTP or FTP server (http(s)://, ftp://).